src/main/java/org/mockito/internal/MockitoCore.java

```diff
@@ -47,26 +46,24 @@
 public class MockitoCore {
 
     private final MockUtil mockUtil = new MockUtil();
-    private final MockingProgress mockingProgress = new ThreadSafeMockingProgress();
```

Comment:
Why not using using `currentThreadMockingProgress`, it is not a static field, and MockitoCore is not serializable anyway